server/src/main/java/pl/edu/agh/utils/LogArchive.java

```diff
@@ -41,4 +41,18 @@ public void commitEntry(LogEntry entry) {
         pendingEntries.entrySet().removeIf(e -> e.getKey().getId() == entry.getId());
         commitedEntries.add(entry);
     }
+
+    public int getLasLogIdx() {
+        return pendingEntries.size() + commitedEntries.size();
```

Comment:
Should be ;) I don't remove entries anywhere. I could look and ID of the last one added, but that's simpler ;)